build/ci/evergreen.yml

```diff
@@ -1008,6 +1008,26 @@ tasks:
         vars:
           E2E_TAGS: opsmanager,generic
           OM_VERSION: 4.4
+  - name: ops_manager_4_4_iam_e2e
+    tags: [ "e2e","ops-manager-44" ]
+    depends_on:
+      - name: compile
+        variant: "code_health"
+        patch_optional: true
+    commands:
+      - func: "clone"
+      - func: "install gotestsum"
+      - func: "build"
+      - func: "deploy spawn host"
+      - func: ssh-ready
+      - func: "install ops manager"
+        vars:
+          ARCHIVE: ${ops_manager_4_4_archive}
+      - func: "set-up ops manager"
+      - func: "e2e test"
+        vars:
+          E2E_TAGS: iam_mms_44
```

Comment:
[q] do we need `iam_mms_44` and `iam_mms_50`? any reason to not use the existing `iam`

Reply:
some iam commands are not available on 4.4 or 5.0

Reply:
or we cannot test it with ops manager

Reply:
it will also be interesting to try to rely on an existing test that deploys ops manager already as that's the part that takes longer on these tests and since they can't be flaky with concurrent tests no need to split them so much

Reply:
> some iam commands are not available on 4.4 or 5.0

can we make it then so that it's a combination `iam,om5` and the tests where is not available have `!om5` that way is more obvious they are not supported and not an overlook

Reply:
Done 
> can we make it then so that it's a combination iam,om5 and the tests where is not available have !om5 that way is more obvious they are not supported and not an overlook

I don't have a strong opinion, I believe is easy to see that we are running iam test against mms if we keep them separated. In addition, tests takes only ~12m
> it will also be interesting to try to rely on an existing test that deploys ops manager already as that's the part that takes longer on these tests and since they can't be flaky with concurrent tests no need to split them so much

Reply:
> tests takes only ~12m

the whole test takes 12m, where about 10 of those is installing ops manager, not a blocker right now but those minutes add up